docroot/modules/humsci/hs_basic_page/hs_basic_page.install

```diff
@@ -226,3 +226,66 @@ function hs_basic_page_update_10000() {
     }
   }
 }
+
+/**
+ * Replace paragraphs_browser with default paragraphs widget.
+ */
+function hs_basic_page_update_10001() {
+  $base_settings = [
+    'edit_mode' => 'closed',
+    'closed_mode' => 'summary',
+    'autocollapse' => 'all',
+    'closed_mode_threshold' => 0,
+    'add_mode' => 'modal',
+    'form_display_mode' => 'default',
+    'features' => [
+      'add_above' => 'add_above',
+      'collapse_edit_all' => 'collapse_edit_all',
+      'convert' => '0',
+      'duplicate' => 'duplicate',
+    ],
+  ];
+  $third_party_settings = [
+    'paragraphs_features' => [
+      'add_in_between' => FALSE,
+      'add_in_between_link_count' => 3,
+      'delete_confirmation' => FALSE,
+      'show_drag_and_drop' => TRUE,
+      'show_collapse_all' => TRUE,
+    ],
+    'paragraphs_ee' => [
+      'paragraphs_ee' => [
+        'dialog_off_canvas' => FALSE,
+        'dialog_style' => 'tiles',
+      ],
+    ],
+  ];
+  $fields = [
+    'field_hs_page_components' => 'node.hs_basic_page.default',
+    'field_hs_page_hero' => 'node.hs_basic_page.default',
+    'field_hs_priv_page_components' => 'node.hs_private_page.default',
+  ];
+  $entity_form_display_storage = \Drupal::entityTypeManager()->getStorage('entity_form_display');
+  foreach ($fields as $field => $form_display_id) {
+    /** @var \Drupal\Core\Entity\Display\EntityFormDisplayInterface $form_display */
+    $form_display = $entity_form_display_storage->load($form_display_id);
+    if (empty($form_display)) {
+      continue;
+    }
+    $component = $form_display->getComponent($field);
+    if (empty($component)) {
+      continue;
+    }
+    $component['type'] = 'paragraphs';
+    $title = $component['settings']['title'];
+    $title_plural = $component['settings']['title_plural'];
+    $default_paragraph_type = $component['settings']['default_paragraph_type'];
+    $component['settings'] = $base_settings;
+    $component['settings']['title'] = $title;
+    $component['settings']['title_plural'] = $title_plural;
+    $component['settings']['default_paragraph_type'] = $default_paragraph_type;
```

Comment:
```suggestion
  $component_settings = array_intersect_key($component['settings'], array_flip(['title', 'title_plural', 'default_paragraph_type']));
    $settings = array_merge($component_settings, $base_settings);

    $component['type'] = 'paragraphs';
    $component['settings'] = $settings;
```

A couple array functions here could help simplify this.
* https://www.php.net/manual/en/function.array-intersect-key.php
* https://www.php.net/manual/en/function.array-flip.php
* https://www.php.net/manual/en/function.array-merge.php

Reply:
Good idea! Thanks for providing the links to the functions. Fixed now.